File: sum_natural.java
import java.util.*;

class sum_natural {
    public static void main() {
        int n, sum = 0;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the number of natural numbers to be added");
        n = sc.nextInt();
        sum = (n * (n + 1)) / 2;
        System.out.println("The sum of first " + n + " natural numbers is " + sum);
        sc.close();
    }
}

File: sum_natural.c
#include <stdio.h>
int main() {
    int n, sum = 0;
    printf("Enter the number of natural numbers to be added : ");
    scanf("%d", &n);
    sum=(n*(n+1))/2;
    printf("The sum of first %d natural numbers is : %d", n, sum);
    return 0;
}